src/pages/home/songs/SongContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box } from "@mui/material";
+import Popup from "components/Popup";
+import { History } from "history";
+import { useState } from "react";
+import SongGrid from "./SongGrid";
+import SongUploadForm from "./SongUploadForm";
+export interface SongsProps {
+  history: History;
+}
+
+const Songs = ({ history }: SongsProps) => {
```

Comment:
Idealy, the file name and component name should be the same, just to keep it obvious what component is being exported.

I'm also hesitant to call it `SongContainer`, as historically a `Container` in React refers to a component that connects the app state to another component. A new developer may see the component name and assume that is what it is.

With functional components and Redux, Redux suggests using hooks to reference the app state in multiple components, instead of having a single container. I think just keeping the name the same as the page (`Songs`) probably works better.